modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -1019,17 +1021,35 @@ const FDSEntries = ({
 					{
 						data: {
 							id: 'view',
+							permissionKey: 'get',
 						},
 						icon: 'view',
 						label: Liferay.Language.get('view'),
 						onClick: onViewClick,
 					},
 					{
+						data: {
+							permissionKey: 'update',
+						},
 						icon: 'pencil',
 						label: Liferay.Language.get('rename'),
 						onClick: onRenameClick,
 					},
 					{
+						data: {
+							permissionKey: 'permissions',
+							size: 'full-screen',
+							title: Liferay.Language.get('permissions'),
+						},
+						href: permissionsURL,
+						icon: 'password-policies',
+						label: Liferay.Language.get('permissions'),
```

Comment:
Do we need both `label` and `data.title`?

Reply:
Yes, we'd need both since `data.title` is used for the modal title, so they could be different in some cases. https://github.com/liferay-frontend/liferay-portal/blob/db3961b010675330667a5926b9437719c024c74f/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js#L99